Vagrantfile

```diff
@@ -53,6 +53,7 @@ Vagrant.configure(2) do |config|
       ansible-playbook -u vagrant -i 'localhost,' --extra-vars "aws_profile=raster-foundry" \
           raster-foundry.yml
       cd /opt/raster-foundry
+      export AWS_PROFILE=raster-foundry
```

Comment:
Yeah, that was prompted the switch to `.bashrc` so I could try and just `source /home/vagrant/.bashrc` here, but that didn't seem to work.